spec/index.bs

```diff
@@ -1249,17 +1249,9 @@ partial interface mixin WindowOrWorkerGlobalScope {
 </pre>
```

Comment:
Line 1243: not part of this patch, but it seems necessary to understand and document it. Why is `or Function` acceptable here? This seems to be a hole in TT.

Reply:
Because a function isn't dangerous?
It's just so you can do like setTimeout(() =>  delayedThing(), 500);

Reply:
> Because a function isn't dangerous? It's just so you can do like setTimeout(() => delayedThing(), 500);

What if `x` is user-controlled:

```
let f = () => { console.log(x ? "do good" : "do bad" )}
```

Reply:
Right, TT can't prevent that and the problem is unrelated to `setTimeout`. Feel free to resolve this conversation.